testdata/baselines/reference/fourslash/hover/DeprecatedInheritedJSDocOverload.baseline

```diff
@@ -41,12 +41,12 @@
         "character": 11
       },
       "Name": "",
-      "Data": null
+      "Data": {}
     },
     "item": {
       "contents": {
         "kind": "markdown",
-        "value": "```tsx\n(method) ThingWithDeprecations.subscribe(observer?: PartialObserver\u003cvoid\u003e): Subscription\n```\n"
+        "value": "```tsx\n(method) ThingWithDeprecations.subscribe(observer?: PartialObserver<void>): Subscription\n```\n"
```

Comment:
This has no effect on clients who decode this just fine, but the HTML escaping junk going away means our baseline looks nicer.